src/openzaak/components/catalogi/admin/mixins.py

```diff
@@ -410,3 +417,25 @@ def get_concept(self, obj):
         if not obj:
             return True
         return obj.zaaktype.concept
+
+
+class NotificationMixin:
+    def save_model(self, request, obj, form, change):
+        if isinstance(obj, ZaakType):
+            viewset = ZaakTypeViewSet()
+        elif isinstance(obj, InformatieObjectType):
+            viewset = InformatieObjectTypeViewSet()
+        elif isinstance(obj, BesluitType):
+            viewset = BesluitTypeViewSet()
+
+        scheme = api_settings.DEFAULT_VERSIONING_CLASS()
+        request.version, request.versioning_scheme = (
+            scheme.determine_version(request, version=api_settings.DEFAULT_VERSION),
+            scheme,
+        )
+
+        viewset.action = "update"
+        data = viewset.serializer_class(obj, context={"request": request}).data
+        viewset.notify(status_code=200, data=data, instance=obj)
```

Comment:
You shouldn't always send a notification, only when something has changed. Any chance you can figure that out? Seems like it should be possible given that you have the `form` object and the `obj` instance.

Also - this doesn't account for `create` notifications rather than `update`.